invenio_records_rest/views.py

```diff
@@ -564,6 +575,10 @@ def post(self, **kwargs):
 
         db.session.commit()
 
+        # Index the record
+        if self.indexer_class:
+            self.indexer_class().index_by_id(record_uuid)
```

Comment:
I think it's better to use ``self.indexer.index(record)`` because ``index_by_id()`` will make a query to the database which is not needed, since you already have the record available.